spec/app1_spec.rb

```diff
@@ -61,5 +61,9 @@
         expect(chef_run).to create_systemd_service(s)
       end
     end
+
+    describe package('postgresql-devel') do
+      it { should be_installed }
+    end
```

Comment:
This is a serverspec test and needs to go within its own file in `test/integration`